modules/sync/protocolbuffers/gofeatures/cas/8fd51058773ac1c2ee378a04f3776fb374ae29813af9b41eac55373543f62effe422b29e76fdd41c0f540fdebe632794cc5b69f2b7a4226b9a1ce63f563a339f

```diff
@@ -0,0 +1,19 @@
+**This is a third-party repository managed by Buf.**
+
+This module contains Protobuf sources that are defined in the
+https://github.com/protocolbuffers/protobuf-go repo. This is currently
+limited to custom features used with [Editions](https://protobuf.dev/editions/overview/)
+to provide perfect backwards compatibility in generated Go code when
+migrating "proto2" sources to Editions.
+
+Updates to the [source repository](https://github.com/google/cel-spec) are automatically synced on a
```

Comment:
It looks like we need to regenerate these files - this doesn't match the buf.md above as it includes content from `cel-spec`.

Reply:
Doh, my bad. I fixed a copy+pasta bug in the sources and forgot to re-generate 🤦